zookeeper/versions.py

```diff
@@ -1,14 +1,8 @@
 versions = [
-    {
-        "product": "3.8.4",
-        "java-base": "11",
-        "java-devel": "11",
-        "jmx_exporter": "1.0.1",
-    },
     {
         "product": "3.9.2",
-        "java-base": "11",
-        "java-devel": "11",
+        "java-base": "11",  # Builds fail on Java 17, so it can stay as is.
```

Comment:
You could try compiling it on 11 and running it on 17 (we did that in the past for other products)

Reply:
Good point, I'll give it a try.

Reply:
I tested based on b8487a3cca3c735201b789e2ea54845671d61ea6, and smoke-latest passed.

```bash
zookeeper-operator $ ./scripts/run-tests --test-suite smoke-latest
```

```
--- PASS: kuttl (106.60s)
    --- PASS: kuttl/harness (0.00s)
        --- PASS: kuttl/harness/smoke_zookeeper-3.9.2_use-server-tls-false_use-client-auth-tls-false_openshift-false (106.59s)
PASS
```

Would you consider this good enough?

Reply:
I'll consider this resolved so we can move forward. Let me know if you don't think this is an adequate proof and we can work on it in a new PR.

Reply:
That works for me, thanks!